apps/designer/app/canvas/shared/styles.ts

```diff
@@ -180,6 +180,12 @@ export const useCssRules = ({
       for (property in dynamicStyle) {
         rule.styleMap.set(property, dynamicStyle[property]);
       }
+      // delete previously rendered properties when reset
+      for (const property of rule.styleMap.keys()) {
+        if (dynamicStyle[property] == null) {
```

Comment:
it can never be null or can it? I saw in types it can't but reading this it looks like it can :)

Reply:
I never use `in` operator :)

Reply:
I think using === undefined is even better here